ape_hardhat/providers.py

```diff
@@ -361,19 +361,23 @@ def send_transaction(self, txn: TransactionAPI) -> ReceiptAPI:
         return receipt
 
     def get_transaction_trace(self, txn_hash: str) -> Iterator[TraceFrame]:
-        """
-        Provide a detailed description of opcodes.
-
-        Args:
-            txn_hash (str): The hash of a transaction to trace.
-
-        Returns:
-            Iterator(TraceFrame): Transaction execution trace object.
-        """
         frames = self._make_request("debug_traceTransaction", [txn_hash]).structLogs
         for frame in frames:
             yield TraceFrame(**frame)
 
+    def get_call_tree(self, txn_hash: str) -> CallTreeNode:
+        receipt = self.get_transaction(txn_hash)
+        root_node_kwargs = {
+            "gas_cost": receipt.gas_used,
+            "gas_limit": receipt.gas_limit,
+            "address": receipt.receiver,
+            "calldata": receipt.data,
+            "value": receipt.value,
+            "call_type": CallType.CALL,
+            "failed": receipt.failed,
+        }
+        return get_calltree_from_geth_trace(receipt.trace, **root_node_kwargs)
```

Comment:
The full trace is used here in `get_call_tree` @fubuloubu 
Hardhat does not support the parity trace, so this is what it uses.